fbw-a380x/src/systems/instruments/src/PFD/SpeedIndicator.tsx

```diff
@@ -957,6 +957,59 @@ class V1Offtape extends DisplayComponent<{ bus: EventBus }> {
   }
 }
 
+class ArsBar extends DisplayComponent<{ bus: ArincEventBus }> {
+  private arsPath = Subject.create<string>('');
+
+  private arsVisiblity = Subject.create<string>('hidden');
```

Comment:
For our newer components, we try to move towards working with ConsumerSubjects and MappedSubjects, instead of using setXxx() functions called after subject updates.
Could you try implementing this? 
Examples are in e.g. https://github.com/flybywiresim/aircraft/blob/master/fbw-a380x/src/systems/instruments/src/EWD/elements/EGT.tsx#L25

For example:
```
private readonly slatsFlapsStatus = Arinc429ConsumerSubject.create(this.sub.on('slatsFlapsStatus').whenChanged());
private readonly conf1FSelected = this.slatsFlapsStatus.map((s) => s.getBitValueOr(28, false) && s.getBitValue(29) && s.getBitValue(18) && !s.getBitValue(26));
private readonly arsVisibility = this.conf1FSelected.map((c) => c ? 'visible' : 'hidden');
```

and so on, feel free to DM me for questions

Reply:
Typescript can also infer the type just fine here, so you don't need to specify it with <string>.